### PR TITLE
Properly handle symlink

### PR DESCRIPTION
**Pull request by ethauvin**
So that `kobaltw` can be used as a symlink. eg:

```
ln -s /usr/local/share/kobalt/bin/kobaltw /usr/local/bin/
```
or even has a local installation:

```
ln -s ~/kobalt-1.0.20/bin/kobaltw ~/bin
```

No need to mess around with temporarily exporting paths, etc. I suspect `brew` does something similar.

With that in mind, the manual installation instructions could be changed to:

```
cd yourLocation
unzip kobalt-xxx.zip
cd kobalt-xxx
ln -s bin/kobaltw /usr/local/bin
```
or something of the sort.

### CODE REVIEW

**Comment by cbeust** [comment]
Did you test this on Windows (cygwin, etc...)?

**Comment by ethauvin** [comment]
I thought I did, but then I tested on my laptop and it broke cygwin in some instances. Actually, to be clear, the original script didn't work either. Looking for a fix. Hold on.

Didn't test on OSX, figured you would.

**Comment by ethauvin** [comment]
@cbeust Independently of the `cygwin` issues, that I believe I have a fix for.

Can you test this for me?

```
mkdir tmp
cd tmp
unzip path/to/kobalt-1.0.21.zip
mkdir test
cd test
java -jar ../kobalt-1.0.21/kobalt/wrapper/kobalt-wrapper.jar --init java
```

I've taken out any potential script interference. Notice something?

`kobaltw` is not created?

**Comment by ethauvin** [comment]
@cbeust I figured it out. It only happens, if you already have a `.kobalt` directory. In that case, `kobaltw` and `kobaltw.bat` do not get created. Not sure if that's a `bug` or not. Let me know, I'll look into it, if it is.

Nevertheless, unless it doesn't work on OSX, the symlink fix should be good to go. Tested on cygwin, git bash (msys/mingw), linux, etc.

File: modules/wrapper/src/main/java/com/beust/kobalt/wrapper/Main.java [hunk 1]
```diff
@@ -219,6 +219,20 @@ private Path installDistribution() throws IOException {
                 log(2, "  Couldn't find $VERSION_TEXT, overwriting the installed wrapper");
                 installWrapperFiles(version, wrapperVersion);
             }
+
+            //
+            // Install the launcher if not already found.
+            //
+            File kobaltw = new File(KOBALTW);
+            File kobaltw_bat = new File(KOBALTW_BAT);
```
Comment (cbeust) [inline modules/wrapper/src/main/java/com/beust/kobalt/wrapper/Main.java]: Let's call it `kobaltwBat` to be consistent.

**Pull request merged**